lib/session-store.js

```diff
@@ -0,0 +1,74 @@
+/*
+// https://github.com/expressjs/session#session-store-implementation
+//
+// Uses PL database as a express-session store
+*/
```

Comment:
We don't need to do this immediately, but we should eventually move this to a package like `@prairielearn/session` or `@prairielearn/session-store` so that we can share this with PrairieTest.

Reply:
I was thinking about that, it's one of the reasons the database name is PL-specific. I think the code very much could be shared as a package/library.